scout/parse/panelapp.py

```diff
@@ -90,6 +91,7 @@ def parse_panelapp_panel(
     gene_panel["display_name"] = " - ".join([panel_info["name"], f"[{confidence.upper()}]"])
     gene_panel["institute"] = institute
     gene_panel["panel_type"] = ("clinical",)
+    gene_panel["description"] = f"{PANELAPP_PANELS_URL}{panel_id}"
```

Comment:
😂 Sure, better than before! I had imagined a clickable one. 😜

Reply:
Me too, but it's more complicated than it seems..